src/test/java/com/wikia/webdriver/common/core/urlbuilder/UrlBuilder.java

```diff
@@ -40,6 +40,9 @@ public String getUrlForWiki(String wikiName) {
 				prefix = "";
 			}
 			suffix = ".com/";
+		} else if (wikiName.endsWith(".de")) {
```

Comment:
Thanks for comment. 
I've seen only such situation for German wikis, so if I would see another case I implement it with regex.